### DISCUSSION TITLE
How to save a flow?

### DISCUSSION THREAD

**Discussion by swamichandra** [langflow-ai/langflow#510] [Q&A]
How do I save a flow? Not export and reimport method.

**Comment by rodrigosnader** [comment]
Hope this is solved in the new version? :)

**Comment by carlosrcoelho** [comment]
Hey!

Thanks for sharing your question here. I'm sorry for not getting back to you sooner.
Langflow has gone through some major changes since then. I recommend trying out the new version for the best experience. If you have any issues, please open a new one; we'll be here to help.
